### ISSUE TITLE
1

### ISSUE THREAD

**Issue by KonstantinusPrime** [ocornut/imgui#5196]
1

**Comment by ocornut** [comment]
As you can see in your screenshot the font isn't used, so maybe you aren't loading the font at the right time?
Use Metrics->Fonts to inspect what is actually loaded.

**Comment by KonstantinusPrime** [comment]
Thanks for that fast reply!
I can confirm, even when forcing the font, the issue is still persistent:
![grafik](https://user-images.githubusercontent.com/103606826/163287984-13812123-505d-466d-bfe9-3082acdaa34c.png)

**Comment by ocornut** [comment]
Well the glyphs are not loaded. Use a debugger to step into code during Build() and find why.
Might need freetype to support this exact font? I don't know. (see #4566 #4202 #3369 where it works)

**Comment by KonstantinusPrime** [comment]
I played some more with it and came to the following result while using your advice and changed to freetype as well as tried the following lines:
```c
    io.Fonts->AddFontFromFileTTF("../../misc/fonts/Cousine-Regular.ttf", 16.0f);
    {
        static ImWchar ranges[] = { 0x1, 0x10FFFF, 0 }; // 0x10FFFF should be max unicode range?
        static ImFontConfig cfg;
        cfg.OversampleH = cfg.OversampleV = 1;
        cfg.MergeMode = true;
        cfg.FontBuilderFlags |= ImGuiFreeTypeBuilderFlags_LoadColor; // color support
        io.Fonts->AddFontFromFileTTF("C:\\Windows\\Fonts\\seguiemj.ttf", 16.0f, &cfg, ranges);
    }
```
I get 1963 glyphs which seems to be mostly emojis: 
![grafik](https://user-images.githubusercontent.com/103606826/163453176-034de378-e969-481d-893c-39e1bc1728c6.png)

now, would it be possible to add said missing glyphs myself or would I need to use another font? I already tried ArialUni, but it gave me like 30k chinese glyphs only, but not the ones I search.
Basically I tried it with the character: https://www.compart.com/de/unicode/U+1D5D4

Thank you so much, I really appreciate it!

**Comment by KonstantinusPrime** [comment]
nvm. got it all sorted out by using another font which supports said ranges using the fontsupport look-up: https://www.fileformat.info/info/unicode/char/1d5d4/fontsupport.htm

Thank you for all your input!
